src/components/pager/index.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component, Fragment } from 'react';
+import classNames from 'classnames';
+import { number, bool, func } from 'prop-types';
+
+export default class Pager extends Component {
+  static propTypes = {
+    current: number.isRequired,
+    total: number.isRequired,
```

Comment:
Note: Drupal core actually doesn't use "total" anymore, because the DB query for that is quite slow. Maybe we should switch to a "next" pager, just like ```/admin/content``` has it already.

Reply:
Humm, yep. That's probably a bit easier to implement. Maybe we can offer both versions.